scripts/astrokat-targets.py

```diff
@@ -28,111 +28,134 @@
 except ImportError:  # not a processing node
     text_only = True
 
+# hard coded globals
+caltag_dict = {
+    "bp": "bandpass",
+    "delay": "delay",
+    "flux": "flux",
+    "gain": "gain",
+    "pol": "polarisation",
+}
+cal_tags = caltag_dict.keys()
+
 
 def cli(prog):
     """Define command line input arguments."""
     usage = "{} [options]".format(prog)
-    description = "Calibrator selection for MeerKAT telescope"
+    description = 'calibrator selection for MeerKAT telescope'
 
     parser = argparse.ArgumentParser(
         usage=usage,
         description=description,
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-    )
-    parser.add_argument("--version", action="version", version=__version__)
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=__version__)
     parser.add_argument(
         "--pi",
         type=str,
-        help="Name of principal investigator or designated project leader",
+        help="name of principal investigator or designated project leader",
     )
     parser.add_argument(
         "--contact",
         type=str,
         help="PI contact details, such as email address or phone number",
     )
-    parser.add_argument("--prop-id", type=str, help="Proposal ID")
-    cal_tags = ["gain", "bp", "flux", "pol", "delay"]
+    parser.add_argument(
+        "--prop-id",
+        type=str,
+        help="proposal ID")
     parser.add_argument(
         "--cal-tags",
         type=str,
         nargs="+",
         metavar="<tag>",
         choices=cal_tags,
-        help="List of tags specifying types of calibrators to provide: "
-        "gain bp flux pol",
+        help="list of tags specifying types of calibrators to provide: "
+             "{}".format(' '.join(cal_tags)),
```

Comment:
Suggest using `sorted(cal_tags)` for readability - the list comes from a dict, so unordered (at least on Python 2)